jOOR-java-8/src/main/java/org/joor/Compile.java

```diff
@@ -195,6 +196,11 @@
         public OutputStream openOutputStream() {
             return os;
         }
+
+        @Override
+        public CharSequence getCharContent(final boolean ignoreEncodingErrors) {
+            return new String(this.os.toByteArray(), Charset.forName("UTF-8"));
```

Comment:
Probably the encoding can be obtained somewhere else.

Reply:
Btw, ```Charset.forName("UTF-8")``` can be replaced with ```StandardCharsets.UTF_8```.

Reply:
```
@since 1.7
```

We still support Java 6

Reply:
... not in that class, though. So yes, it could be replaced. Will do